js/live-editor.js

```diff
@@ -132,37 +132,6 @@ window.LiveEditor = Backbone.View.extend({
             type: this.editorType
         });
 
-        var tooltipEngine = this.config.editor.tooltipEngine;
```

Comment:
This is the feature from Gigabyte-Giant that was added when live completion was still enabled, to let people turn it off.